sys/include/net/ble.h

```diff
@@ -0,0 +1,405 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ble General BLE defines
+ * @ingroup     net
+ * @brief       General values defined by the BT standard
+ * @{
+ *
+ * @file
+ * @brief       General BLE values as defined by the BT standard
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NET_BLE_H
+#define NET_BLE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    BT version constants
```

Comment:
Bluetooth

Reply:
Mainly, because just BT in a heading looks weird IMHO:

![image](https://user-images.githubusercontent.com/675644/38367870-198ebc94-38e4-11e8-8ae3-eebfe17f0d3c.png)

Reply:
OK